tasks/src/table-question-answering/data.ts

```diff
@@ -0,0 +1,68 @@
+import type { Task DataCustom } from "../Types";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{ 
+			description:"The Fewshot Table dataset consists of tables that naturally occur on the web, that are formatted as few-shot tasks for fine-tuning language models to improve their few-shot performance.",
+			id: "JeremyAlain/123_test",
```

Comment:
Can we put a maintained dataset as I see the latest issue was opened 3 months ago and hasn't yet resolved.